src/main/java/edu/harvard/iq/dataverse/GlobalId.java

```diff
@@ -25,6 +27,8 @@
     public static final String HDL_PROTOCOL = "hdl";
     public static final String HDL_RESOLVER_URL = "https://hdl.handle.net/";
     public static final String DOI_RESOLVER_URL = "https://doi.org/";
+    public static final String DISALLOWED_CHARACTERS = ".*[<>()].*";
```

Comment:
This can safely be removed as we are just whitelisting

Reply:
Thought I took that out... but that's what code review is for, right?

Reply:
Totally! One more thing, we should probably put the blacklist string into the bundle.properties file. It helps a bit with later maintenance and customization